.github/workflows/go-verdiff.yaml

```diff
@@ -1,9 +1,6 @@
 name: go-verdiff
 on:
   pull_request:
-    paths:
-    - '**.mod'
-    - '.github/workflows/go-verdiff.yaml'
```

Comment:
We should only run it when we change the go,mod and the action itself right?

Reply:
The problem is that GHA conditional stuff by path doesn't seem to align with prow/tide's idea of optional. So when there aren't changes in the path tripwires, tide waits forever for status that will never come (since the GHA doesn't run).

Reply:
I would personally prefer that we find a way to make prow better-informed about conditionality so that we could continue doing targeted CI upstream, which generally makes sense.

Reply:
It's causing problems when it's a required test; it doesn't provide any results. So, we need to have it run all the time if we want to make it a required test.

Reply:
In other words, "This is a required test, but oh, I'm only supposed to run it when these files are changed." So, unless those file are changed, it doesn't run, doesn't generate any results, and thus the CI hangs waiting for it.